front/_check_webserver_config.php

```diff
@@ -36,6 +36,6 @@
     // `Glpi\Kernel\Kernel` class will exists if the request was processed by the `/public/index.php` file,
     // and will not be found otherwise.
     header('HTTP/1.1 404 Not Found');
-    readfile(__DIR__ . '/../index.html');
+    Safe\readfile(__DIR__ . '/../index.html');
```

Comment:
Thez autoloader will not be available here. this should be reverted and the error ignored.